tests/test_generate_database_data.py

```diff
@@ -70,40 +71,53 @@ def test_set_all_frameworks_to_expired(self):
         self.api_client.update_framework.assert_any_call("g-cloud-7", {"status": "expired"})
 
 
-class TestCreateGCloudFramework(TestGenerateDataBase):
+class TestOpenGcloud12(TestGenerateDataBase):
```

Comment:
```suggestion
class TestOpenGCloud12(TestGenerateDataBase):
```

Minor nit - 'G-Cloud' gets Camel-cased to `GCloud`

Reply:
Make sense.